go-controller/pkg/ovn/multicast_test.go

```diff
@@ -280,6 +280,104 @@ var _ = ginkgo.Describe("OVN NetworkPolicy Operations with IP Address Family", f
+				app.Action = func(ctx *cli.Context) error {
+					longNameSpace1Name := "abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijk" // create with 63 characters
+					namespace1 := *newNamespace(longNameSpace1Name)
+					longNameSpace2Name := "abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijl" // create with 63 characters
+					namespace2 := *newNamespace(longNameSpace2Name)
+					nPodTestV4 := newTPod(
+						"node1",
+						"10.128.1.0/24",
+						"10.128.1.2",
+						"10.128.1.1",
+						"myPod1",
+						"10.128.1.3",
+						"0a:58:0a:80:01:03",
+						namespace1.Name,
+					)
+					nPodTestV6 := newTPod(
+						"node1",
+						"fd00:10:244::/64",
+						"fd00:10:244::2",
+						"fd00:10:244::1",
+						"myPod2",
+						"fd00:10:244::3",
+						"0a:58:dd:33:05:d8",
+						namespace1.Name,
+					)
+					var tPods []testPod
+					var tns1PodIPs []string
+					if m.IPv4Mode {
+						tPods = append(tPods, nPodTestV4)
+						tns1PodIPs = append(tns1PodIPs, nPodTestV4.podIP)
+					}
+					if m.IPv6Mode {
+						tPods = append(tPods, nPodTestV6)
+						tns1PodIPs = append(tns1PodIPs, nPodTestV6.podIP)
+					}
+
+					var pods []v1.Pod
+					for _, tPod := range tPods {
+						pods = append(pods, *newPod(tPod.namespace, tPod.podName, tPod.nodeName, tPod.podIP))
+					}
+
+					fakeOvn.startWithDBSetup(initialDB,
+						&v1.NamespaceList{
+							Items: []v1.Namespace{
+								namespace1,
+								namespace2,
+							},
+						},
+						&v1.PodList{
+							Items: pods,
+						},
+					)
+					setIpMode(m)
+
+					for _, tPod := range tPods {
+						tPod.populateLogicalSwitchCache(fakeOvn, getLogicalSwitchUUID(fakeOvn.controller.nbClient, "node1"))
+					}
+
+					err := fakeOvn.controller.WatchNamespaces()
+					gomega.Expect(err).NotTo(gomega.HaveOccurred())
+					fakeOvn.controller.WatchPods()
+					ns1, err := fakeOvn.fakeClient.KubeClient.CoreV1().Namespaces().Get(context.TODO(), namespace1.Name, metav1.GetOptions{})
+					gomega.Expect(err).NotTo(gomega.HaveOccurred())
+					gomega.Expect(ns1).NotTo(gomega.BeNil())
+					ns2, err := fakeOvn.fakeClient.KubeClient.CoreV1().Namespaces().Get(context.TODO(), namespace2.Name, metav1.GetOptions{})
+					gomega.Expect(err).NotTo(gomega.HaveOccurred())
+					gomega.Expect(ns2).NotTo(gomega.BeNil())
+
+					// Enable multicast in the namespace
+					mcastPolicy := multicastPolicy{}
+					portsns1 := []string{}
+					for _, tPod := range tPods {
+						portsns1 = append(portsns1, tPod.portUUID)
+					}
+
+					expectedData := mcastPolicy.getMulticastPolicyExpectedData(longNameSpace1Name, portsns1)
+					acl := expectedData[0].(*nbdb.ACL)
+					gomega.Expect(acl.Name).NotTo(gomega.BeNil())
+					// since ACL names only use suffix, name will always be unique for multicast as only the suffix will get cropped
+					gomega.Expect(*acl.Name).To(gomega.Equal(longNameSpace1Name))
```

Comment:
ack (I thought my existing comments were ok but seems there aren't, :) will add more context.)

Reply:
since ACLIndexes has merged and we have removed "names" for multicast ACL's alltogether from what I can see' we don't need this comparison anymore...removed it.